DESIGN-SPEC.md

```diff
@@ -0,0 +1,78 @@
+
+The site should aim for WCAG compliance with respect to text contrast. This can be achieved mainly by overriding Bootstrap as little as possible. Custom colors should be run through Adobe's public [accessible palette picker](https://color.adobe.com/create/color-accessibility) to ensure differentiation between backgrounds and text.
+
+
+### Content consistency
+
+All site content below the nav bar should have a parent div with the Bootstrap `container` class.
+
+Content should generally be left aligned, except for content in banners such as the site welcome and forum welcome.
+
+The site's color theming revolves around deep purple and light blue accent colors, with black text against light grey backgrounds on the light theme, and light white text against dark grey backgrounds on the dark theme with no or little change to the accent colors. Exact values to be determined per adjustments for accessibility and reduction of potential duplication in the `colors` partial.
+
+The purple accent color is intended for the navbar background and for button controls that don't change pages. The blue accent color is intended for links and button controls that change pages. These are also useful to separate functional groups of buttons.
+
+Headings currently are inconsistent, a possible direction could be to use lighter info blue backgrounds from Bootstrap with black text on the light theme, and the inverted info (a deep navy blue) with white text on the dark theme.
+
+Per [issue #399](https://github.com/adelikat/tasvideos/issues/399), developers seem to be in agreement that usage of overflow scrolls should be avoided where possible. For Y scrolls many of theme are simply not needed anymore with the current site design. Some X scrolls are currently used to hotfix usage of `table` elements which were overflowing mobile widths, the table elements are planned to be replaced by grid or flexbox.
+
+Clickable buttons that act as links should be visibly differentiable from informational elements like titles. This may be possible simply by removing our override of bootstrap's text-decoration: underline on anchor elements as we don't use underline elsewhere on the site, but in any event link buttons should have some visual separation besides just usage of the accent theme colors. These buttons should also maintain the default border radius set by bootstrap for a consistent rounding look across the site.
+
+Some custom icons are used for an 8 bit retro theme, for example the movie tier labels including the star, moon, and verification icons. these icons have an additional 2x and 4x exact prescale, enabling browsers to pick the appropriate image size to get crisp 8 bit edges with the use of responsive image sourcesets. All other icons needed should be pulled from the third party Font Awesome library which is included across the site (version 4.7.0).
```

Comment:
We should remove the version number of FA.  Otherwise we have to update this everytime we upgrade, and the specific version is not essential to the meaning